x/dex/keeper/verification.go

```diff
@@ -9,77 +9,20 @@ import (
 )
 
 func (k Keeper) DepositVerification(goCtx context.Context, msg types.MsgDeposit) (string, string, sdk.AccAddress, []sdk.Int, []sdk.Int, error) {
```

Comment:
Are we not getting rid of this?

Reply:
moved sorting and msg validation call to msg server